ZUGFeRD-Test/ZUGFeRD20Tests.cs

```diff
@@ -422,6 +422,8 @@ public void TestWriteAndReadExtended()
                 attachmentBinaryObject: data,
                 filename: filename2);
 
+            desc.BusinessProcess = "A1";
```

Comment:
can you please create dedicated test cases instead of wiring it into the existing test cases?

Reply:
I made a modification in this sense but I don't know if it will suit you.